Frontend/src/components/FunctionDetail/CitationsTable.js

```diff
@@ -1,4 +1,4 @@
-import './table.css'
+import './../../App.css'
```

Comment:
You do not have to import the App.css in all the files, just importing it in the index.js file should be sufficient.

Reply:
I have updated accordingly